Initial Migration Files Missing in Binary Build

### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### Homebox Version

0.9.1

### What is the issue you are experiencing?

when running
`>.\homebox.exe`

```
panic: open migrations\20220929052825_init.sql: file does not exist

goroutine 1 [running]:
main.main()
        /home/runner/work/homebox/homebox/backend/app/api/main.go:55 +0x72
```

### How can the maintainer reproduce the issue?

download [homebox_Windows_x86_64.zip](https://github.com/hay-kot/homebox/releases/download/v0.9.1/homebox_Windows_x86_64.zip) extract and run homebox.exe

### Deployment

Other

### Deployment Details

Windows 22H2 (10.0.19045.2846)

--- a/backend/internal/data/migrations/migrations.go
+++ b/backend/internal/data/migrations/migrations.go
@@ -3,7 +3,7 @@ package migrations
 import (
 	"embed"
 	"os"
-	"path/filepath"
+	"path"
 )
 
 //go:embed all:migrations
@@ -28,12 +28,12 @@ func Write(temp string) error {
 			continue
 		}
 
-		b, err := Files.ReadFile(filepath.Join("migrations", f.Name()))
+		b, err := Files.ReadFile(path.Join("migrations", f.Name()))
 		if err != nil {
 			return err
 		}
 
-		err = os.WriteFile(filepath.Join(temp, f.Name()), b, 0o644)
+		err = os.WriteFile(path.Join(temp, f.Name()), b, 0o644)
 		if err != nil {
 			return err
 		}

